<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Paint</class>
 <widget class="QMainWindow" name="Paint">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Paint</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>90</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QComboBox" name="m_comboBox_Tools">
         <item>
          <property name="text">
           <string>Pencil</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Brush</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Text</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Eraser</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="m_comboBox_Width">
         <item>
          <property name="text">
           <string>4 px</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>8 px</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>16 px</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>32 px</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="m_comboBox_Figures">
         <item>
          <property name="text">
           <string>Line</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Curve</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Ellipsoid</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Square</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Polygon</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="5">
        <widget class="QPushButton" name="m_pB_ColorPalette">
         <property name="text">
          <string>Palette</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QPushButton" name="m_pB_Color1">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLineEdit" name="m_TextToolLineEdit">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="m_pB_plus">
         <property name="text">
          <string>Приблизить</string>
         </property>
        </widget>
       </item>
       <item row="1" column="5">
        <widget class="QPushButton" name="m_pB_Color2">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="m_pB_minus">
         <property name="text">
          <string>Отдалить</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2" colspan="2">
        <widget class="QLabel" name="m_label_CurrentTool">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionClose"/>
    <addaction name="actionCreate_new_file"/>
    <addaction name="actionOpen_file"/>
    <addaction name="actionSave"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionCreate_new_file">
   <property name="text">
    <string>Create new file</string>
   </property>
  </action>
  <action name="actionOpen_file">
   <property name="text">
    <string>Open file</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
